### DISCUSSION TITLE
Local Authentication Disabled in login

### DISCUSSION THREAD

**Discussion by rakeshdavid** [requarks/wiki#4610] [Help / Questions] [answered]
Hi,

Is there a way to enable Local Auth via URL or other means? I disabled local auth and locked myself out of Admin mode.

I intend to disable local auth, but I need an admin user. 

Context:
While implementing AUTH0, I wanted to disable local login without realizing that my Auth0 account is not an Admin. I need to make my AUTH0 account admin, but I am not sure I found any answers.

Any help is much appreciated.

RD

**Comment by ghost** [comment, marked as answer]
In general, you can do the following:

1.  Define your OAUTH0 and local authentication provider, and keep both activated. 
2. Go to Admin Settings -> Security -> Login -> Enable 'Hide Local Authentication Provider'
3. Apply settings
4. After logging out, you should see only your OUATH0 provider while login in again. 
5. To get access to the local authentication provider, you should use `https://mywiki.domain/login?all`. As a result, you will be redirected to the login page with the local authenticator along with your OAUTH0 provider.

**Reply by rakeshdavid** [reply]
@dku-xyt Thank you so much, that worked! I appreciate the help.

**Reply by Jonty16117** [reply]
Hi, I am having the same problem, but the `https://mywiki.domain/login?all` url is still redirecting me to the Auth0 page, can you please help me?

**Reply by MrWormsy** [reply]
Hi, there is still the issue depicted by @Jonty16117 in the latest version.
